specification/datashare/resource-manager/readme.go.md

```diff
@@ -23,4 +24,13 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 
 ``` yaml $(tag) == 'package-2018-11-01-preview' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/datashare/mgmt/2018-11-01-preview/$(namespace)
+```
+
+### Tag: package-2019-11-01 and go
+
+These settings apply only when `--tag=package-2019-11-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2019-11-01' && $(go)
+output-folder: $(go-sdk-folder)/services/stable/datashare/mgmt/2019-11-01/$(namespace)
```

Comment:
Please change this to
`$(go-sdk-folder)/services/datashare/mgmt/2019-11-01/$(namespace)`

Reply:
@ArcturusZhang  updated. thanks.